lib/pure/uri.nim

```diff
@@ -343,7 +342,7 @@ proc combine*(uris: varargs[Uri]): Uri =
   ## **See also:**
   ## * `/ proc <#/,Uri,string>`_ for building URIs
   runnableExamples:
-    let foo = combine(parseUri("https://nim-lang.org/blog.html"), parseUri("/install.html"))
+    let foo = combine([parseUri("https://nim-lang.org/blog.html"), parseUri("/install.html")])
```

Comment:
I agree that the previous example was not showcasing the correct `combine` proc.

But if you want to show the behaviour of `varargs`, maybe something like this would be better:
```nim
let foo = combine(parseUri("https://"), parseUri("nim-lang.org/"), parseUri("about.html"))
```

Reply:
You are totally right. Updated it with 
```nim
let foo = combine(parseUri("https://nim-lang.org/"), parseUri("docs/"), parseUri("manual.html"))
```